docs/restore-from-dump.md

```diff
@@ -0,0 +1,141 @@
+# Restore Stelace API data into own infrastructure
+
+Prerequisite:
+- Fulfill the .env file as indicating in README.md
+- Start your databases and Stelace API server
+- Have the Stelace dump
+
+## Restore PostgreSQL data via dump
+
+```sh
+pg_restore -h <host> --port <port> -d <database_name> -U <user> --no-owner --role=<user> <dump_name>.dump
+```
+
+This will restore data from test and live environments.
```

Comment:
It’s confusing. You need to run this separately for test and live environments if dump does not include them both.

`This will restore data from test and live environments as long as dump includes them both.`

Reply:
You're right. I'll update the sentence.